packages/node_modules/@webex/plugin-meetings/src/networkQualityMonitor/index.js

```diff
@@ -0,0 +1,178 @@
+import EventsScope from '../common/events/events-scope';
+import {EVENT_TRIGGERS, STATS} from '../constants';
+
+
+/**
+  * Meeting - network quality judgment event
+  * Emitted on each interval of retrieving stats Analyzer data
+  * @event network:quality:judgment
+  * @type {Object}
+  * @property {string} mediaType {video|audio}
+  * @property {boolean} networkAcceptable is network overal acceptable
+  * @property {Object} networkQualityDataStructure for each indicator type
+  * @property {boolean} networkQualityDataStructure.acceptable
+  * @property {Number} networkQualityDataStructure.value - score or RTC value for each indicator type
+  * @memberof NetworkQualityMonitor
+  */
+/**
+ * NetworkQualityMonitor class that will emit events based on detected quality
+ *
+ * @class NetworkQualityMonitor
+ * @extends {EventsScope}
+ */
+export default class NetworkQualityMonitor extends EventsScope {
+  /**
+   * Creates a new instance of NetworkQualityMonitor
+   * @constructor
+   * @public
+   * @param {Object} config
+   */
+  constructor(config) {
+    super();
+    this.config = config;
+    this.indicatorTypes = Object.freeze({
+      PACKETLOSS: 'packetLoss',
+      LATENCY: 'latency',
+      JITTER: 'jitter'
+    });
+    this.adequeteExistingNetworkQuality = true;
+    this.networkQualityDataStructure = {};
```

Comment:
Should we name this to something else may be just networkQualityStatus or something ?

Reply:
sure I can if you like - was just being more specific about the name - but it's probably redundant since it's clearly an object

Reply:
updated